src/Controls/tests/DeviceTests/HandlerTestBase.cs

```diff
@@ -77,6 +78,7 @@ public void EnsureHandlerCreated(Action<MauiAppBuilder> additionalCreationAction
 
 			appBuilder.Services.AddSingleton<IDispatcherProvider>(svc => TestDispatcher.Provider);
 			appBuilder.Services.AddScoped<IDispatcher>(svc => TestDispatcher.Current);
+			appBuilder.Services.TryAddSingleton(typeof(IApplication), (_) => new Application());
```

Comment:
Does the generic not exist? Not that it matters at all, but you know, it is not the same as the line above.

```suggestion
			appBuilder.Services.TryAddSingleton<IApplication>(svc => new Application());
```

Reply:
Pushed up using the generic